examples/PinnedH2O/QuenchAndEVP/mgmol_MVP.cfg

```diff
@@ -0,0 +1,36 @@
+verbosity=1
+xcFunctional=PBE
+FDtype=Mehrstellen
+[Mesh]
+nx=64
+ny=64
+nz=64
+[Domain]
+ox=-6.
+oy=-6.
+oz=-6.
+lx=12.
+ly=12.
+lz=12.
+[Potentials]
+pseudopotential=pseudo.O_ONCV_PBE_SG15
+pseudopotential=pseudo.H_ONCV_PBE_SG15
+[Run]
+type=quench
+[Thermostat]
+type=Berendsen
+temperature=1000.
+relax_time=800.
+[Quench]
+max_steps=100
+atol=1.e-8
+[Orbitals]
+initial_type=Random
+initial_width=2.
+nempty=10
+[Restart]
+output_level=4
+output_filename=restart_MVP
+[DensityMatrix]
+solver=MVP
+nb_inner_it=20
```

Comment:
20 is too much in this case. should be 2. this is the number of inner iterations and here there is an outer iteration. when no outer iterations are made, indeed it should be larger